aether-kernel/aether/kernel/api/models.py

```diff
@@ -688,7 +684,7 @@ def clean(self):
 
     def save(self, *args, **kwargs):
         try:
-            self.full_clean()
+            self.clean()
```

Comment:
Why `clean` and not `full_clean`?

https://docs.djangoproject.com/en/2.2/ref/models/instances/#validating-objects

Reply:
`full_clean` revalidates all linked objects, which is already done on the [serializer,](https://github.com/eHealthAfrica/aether/blob/refactor-kernel-entity-post/aether-kernel/aether/kernel/api/serializers.py#L381) causing unnecessary calls to the db.  That is the major strain on the current performance.

Reply:
Could you write a comment explaining why, otherwise someone would change it back in the future :grimacing:

Reply:
:+1: